packages/api/amplify_api_dart/lib/src/graphql/web_socket/services/web_socket_service.dart

```diff
@@ -173,7 +173,7 @@ class AmplifyWebSocketService
           );
         }
 
-        final exception = ApiException(
+        final exception = UnknownException(
```

Comment:
nit: Maybe operation error? Seems like swift uses for similar circumstance https://github.com/aws-amplify/amplify-swift/blob/main/AmplifyPlugins/API/Sources/AWSAPIPlugin/Operation/AWSGraphQLSubscriptionTaskRunner.swift#L195 though not exactly the same.

Reply:
Yes, Unknown should be used very sparingly